tests/lib/cli-engine/cli-engine.js

```diff
@@ -3709,46 +3896,69 @@ describe("CLIEngine", () => {
         describe("don't ignore the entry directory.", () => {
             const root = getFixturePath("cli-engine/dont-ignore-entry-dir");
 
-            it("'executeOnFiles(\".\")' should not load config files from outside of \".\".", () => {
-                CLIEngine = defineCLIEngineWithInMemoryFileSystem({
-                    cwd: () => root,
+            let cleanup;
+
+            beforeEach(() => {
+                cleanup = () => {};
+            });
+
+            afterEach(async () => {
+                await cleanup();
+                shell.rm(path.resolve(root, "../.eslintrc.json"));
```

Comment:
The last test doesn't create `"../.eslintrc.json"`, so `shell.rm` will output something like this when we run `npm test`:

```
rm: no such file or directory: /tmp/eslint/fixtures/cli-engine/.eslintrc.json
```

(observable on CI, too)

Reply:
Okay that's weird, I wasn't seeing this locally. I'll fix it.